plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8453,6 +8453,84 @@ public void testCreateAcidTableUnsupported()
+        String queryId = (String) computeScalar("SELECT query_id FROM system.runtime.queries WHERE query LIKE 'CREATE TABLE " + tableName + "%'");
+        String nodeVersion = (String) computeScalar("SELECT node_version FROM system.runtime.nodes WHERE coordinator");
+        assertQuery(
+                "SELECT * FROM \"test_extra_properties$properties\"",
+                "SELECT 'workaround for potential lack of HIVE-12730', 'false', 'true', '{\n  \"extra.property\" : \"true\"\n}', '0', '0', '" + queryId + "', '" + nodeVersion + "', '0', '0', 'false'");
+        MaterializedResult actualResult = computeActual("SHOW CREATE TABLE " + tableName);
+        String expectedShowCreateTableSql = "CREATE TABLE hive.tpch.test_extra_properties (\n" +
+                "   c1 integer\n" +
+                ")\n" +
+                "WITH (\n" +
+                "   extra_properties = map_from_entries(ARRAY[ROW('extra.property', 'true')]),\n" +
+                "   format = 'ORC'\n" +
+                ")";
+        assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), expectedShowCreateTableSql);
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Test
+    public void testMultipleExtraProperties()
+    {
+        String tableName = format("%s.%s.test_multiple_extra_properties", getSession().getCatalog().get(), getSession().getSchema().get());
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s (\n" +
+                        "  c1 integer\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "  extra_properties = MAP(ARRAY['extra.property.one', 'extra.property.two'], Array['one', 'two'])\n," +
+                        "  format = 'ORC'\n" +
+                        ")",
+                tableName);
+        assertUpdate(createTableSql);
+        String queryId = (String) computeScalar("SELECT query_id FROM system.runtime.queries WHERE query LIKE 'CREATE TABLE " + tableName + "%'");
+        String nodeVersion = (String) computeScalar("SELECT node_version FROM system.runtime.nodes WHERE coordinator");
+        assertQuery(
+                "SELECT * FROM \"test_multiple_extra_properties$properties\"",
+                "SELECT 'workaround for potential lack of HIVE-12730', 'false', 'one', 'two', '{\n  \"extra.property.one\" : \"one\",\n  \"extra.property.two\" : \"two\"\n}', '0', '0', '" + queryId + "', '" + nodeVersion + "', '0', '0', 'false'");
+        MaterializedResult actualResult = computeActual("SHOW CREATE TABLE " + tableName);
+        String expectedShowCreateTableSql = "CREATE TABLE hive.tpch.test_multiple_extra_properties (\n" +
+                "   c1 integer\n" +
+                ")\n" +
+                "WITH (\n" +
+                "   extra_properties = map_from_entries(ARRAY[ROW('extra.property.one', 'one'),ROW('extra.property.two', 'two')]),\n" +
+                "   format = 'ORC'\n" +
+                ")";
```

Comment:
We can use text blocks for these String

Reply:
@Praveen2112 I could not use text blocks for the expected output of `SHOW CREATE TABLE` as they would always trigger the `Whitespace character before closing parenthesis` check style warning. If I updated the string to resolve the check style warning then the test would fail because the string would not match.